### ISSUE TITLE
Helm chart

### ISSUE THREAD

**Issue by richerlariviere** [nats-io/nats-operator#61]
First of all, thank you for your amazing work on Nats and particularly nats-operator.

What do you think of creating a [Helm](https://helm.sh/) chart for nats-operator? That would be nice I think because we could do something like this:

`helm install nats-operator`

I have some knowledge in creating helm charts but my global understanding of nats is not so great so I can help creating a base chart for a minimal setup. Edge cases and more specific uses of nats could be supported later. What do you think?

**Comment by wallyqs** [comment]
A helm chart sounds good, feel free to send the PR for the basic setup :)

**Comment by richerlariviere** [comment]
I have setup a repo here: https://github.com/richerlariviere/nats-cluster-chart. I tested the chart on my cluster and it worked well.

Currently the repo supports client authentication only via password as this is how I use NATS. I plan to add **token support** and the possibility to add other config files like for **cluster authentication**.

In fact, the repo is heavily inspired from the [NATS helm chart](https://github.com/helm/charts/tree/master/stable/nats). Eventually I'd like a configuration parity with the original chart. I also used the manifests in `/example` to build the templates.

When I'm done I'll let you know and we will be able to discuss how you want the chart to be included to the repo (I'm not sure where we could put it, maybe in a `contrib/` directory at the root?).

**Comment by wallyqs** [comment]
Sorry for the wait, sounds good to include in the repo. I'm thinking the following directory structure, wdyt?

```sh
├── helm
│   └── nats-operator
│        ...
│       └── Chart.yaml
```
Thanks for the contribution!

**Comment by richerlariviere** [comment]
That sounds good to me :)

**Comment by wallyqs** [comment]
Fixed via #62
